sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/Transforms.java

```diff
@@ -297,4 +305,73 @@ private static FileReadHeaders toPathReadHeaders(BlobDownloadHeaders h) {
             .setContentCrc64(h.getContentCrc64())
             .setErrorCode(h.getErrorCode());
     }
+
+    static List<BlobSignedIdentifier> toBlobIdentifierList(List<DataLakeSignedIdentifier> identifiers) {
+        if (identifiers == null) {
+            return null;
+        }
+        List<BlobSignedIdentifier> blobIdentifiers = new ArrayList<>();
+        for (DataLakeSignedIdentifier identifier : identifiers) {
+            blobIdentifiers.add(Transforms.toBlobIdentifier(identifier));
+        }
+        return blobIdentifiers;
+    }
+
+    private static BlobSignedIdentifier toBlobIdentifier(DataLakeSignedIdentifier identifier) {
+        if (identifier == null) {
```

Comment:
If this is null do we want to add a null value into the converted list?

Reply:
Since the datalake API just calls into the blob API, we're basically just spitting out exactly what was given to us to pipe it to the corresponding type, so I think its fine to have  that null check